pkgs/by-name/ni/nix-init/package.nix

```diff
@@ -34,10 +34,9 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-0RLEPVtYnwYH+pMnpO0/Evbp7x9d0RMobOVAqwgMJz4=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes."cargo-0.82.0" = "sha256-1G14vLW3FhLxOWGxuHXcWgb+XXS1vOOyQYKVbrJWlmI=";
-  };
+  useFetchCargoVendor = true;
```

Comment:
This option is not available on release-24.11

Sorry for triggered this PR with adding label.